packages/documentation/src/pages/index.tsx

```diff
@@ -602,11 +604,6 @@ function InformationCards() {
   return (
     <div>
       <div className={clsx(styles.container, styles.information_cards)}>
-        <img
-          src={useBaseUrl('/img/Screen_09_background_image.png')}
-          alt=""
-          className={styles.information_cards_background}

```

Comment:
Removing the class breaks the styling. Also the corresponding CSS class is not used anywhere else so we should remove it from the stylesheet.

Reply:
<img width="1243" alt="Screenshot 2024-07-19 at 10 14 23" src="https://github.com/user-attachments/assets/4e9044b4-51b6-4b3b-9882-845afe3581bf">